packages/hoc/src/__tests__/queries/api.test.tsx

```diff
@@ -188,13 +188,15 @@ describe('[queries] api', () => {
                   }
                 })
               })
-              .then(
-                wrap(done, (result: any) => {
+              .then((result: any) => {
+                try {
                   expect(stripSymbols(result.data.allPeople.people)).toEqual(
                     data1.allPeople.people
                   );
-                })
-              );
+                } catch (error) {
+                  done(error);
```

Comment:
Is there any difference between `done(error)` and `done.fail(error)`?

Reply:
No, they're the same (from an API standpoint)